config/locales/time.en_abbrev.yml

```diff
@@ -0,0 +1,8 @@
+en_abbrev:
```

Comment:
This file can be merged with the rest of en-abbrev

Reply:
I proposed wiping out the previous one, in https://github.com/codidact/qpixel/pull/151/commits/f5e7360af5abc02d20cc2cbbc007487eb03af2ea